static/main.js

```diff
@@ -194,6 +194,63 @@ const aliases = {
     "‟": ["''"]
 };
 
+// While the two byte aliases attempt to identify what characters look like,
```

Comment:
Btw, some of the two-byte aliases are also what they do, for example logical OR is `or`.